### ISSUE TITLE
Patch backport request: `kernel32/profile: Trim spaces from section names on load.`

### ISSUE THREAD

**Issue by qwertychouskie** [ValveSoftware/Proton#6359]
# Feature Request

## I confirm:
- [x] that I haven't found another request for this feature.
- [x] that I have checked whether there are updates for my system available that
      contain this feature already.

## Description 
The following patch was recently added to upstream Wine: https://gitlab.winehq.org/wine/wine/-/commit/8d9eb707c61455583a42891d6841ba9feeaf78d3

This patch fixes an issue where INI files with extra spaces in the section names won't load.  This affects the TMUnlimited mod for TrackMania Nations Forever (11020) and TrackMania United Forever (7200).  Before the patch, it was impossible to change settings in the mod, as the INI would always fail to read.  It's very possible that other software could also be affected by this bug, though I don't know of any at the moment.

## Justification [optional] 
Patch is already upstreamed, it would be nice to be able to just use Proton via Steam rather than manually launching the game using stock Wine or waiting until the eventual re-base of Proton.

## Risks [optional]
Basically none

## References [optional] 
https://gitlab.winehq.org/wine/wine/-/commit/8d9eb707c61455583a42891d6841ba9feeaf78d3
https://bugs.winehq.org/show_bug.cgi?id=53814
